packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx

```diff
@@ -18,7 +18,7 @@ export const WfoBadge: FC<WfoBadgeProps> = ({
     ...restProps
 }) => (
     <EuiBadge {...restProps}>
-        <EuiText color={textColor} size={size ?? 'xs'}>
+        <EuiText title="" color={textColor} size={size ?? 'xs'}>
```

Comment:
What does the empty title prop? 
```suggestion
        <EuiText color={textColor} size={size ?? 'xs'}>
```

Reply:
It prevents showing the default browser tooltip. We don't want those showing.